### DISCUSSION TITLE
File that is not a part of any collection

### DISCUSSION THREAD

**Discussion by bykhov** [alshedivat/al-folio#2145] [Q&A] [answered]
I would like to have some standalone md files that are **not** a formal part of any collection (e.g. _posts, _new, etc).
How can it be done and how can they be referenced from inside the site?

**Comment by george-gca** [comment, marked as answer]
Take a look at the jekyll documentation about [pages](https://jekyllrb.com/docs/pages/). Or maybe you want to [create your own collection](https://jekyllrb.com/docs/collections/).

**Reply by bykhov** [reply]
Thanks!
I was not aware it was so simple.
